### ISSUE TITLE
[Intro]:Keyboard focus is not visible on first interactive element on activating the 'Intro' link.[Sev3][WCAG2.4.3]

### ISSUE THREAD

**Issue by SrujanaAnaganti** [microsoft/tsdoc.org-website#19]
**Test Environment:**
OS Build: Windows 10
Browser: Edge Dev
URL: https://tsdoc.org/

**Repro Steps:**
1) Open the above URL and login with valid credentials.
2) TSDoc page will get displayed.
3) Navigate to 'Into' link and activate it and observe keyboard focus issue.

**Actual:**
Keyboard focus is not visible on first interactive element on activating the 'Intro' link rather it is moving to the TSDoc link.

**Expected:**
When opening the dialog or page, focus is expected to move to first interactive element and focus should not leave the modal/dialog/pop-up until it is closed. When a modal/dialog/pop-up is opened, focus should move into that item and when it is closed, focus should return to the same control that opened it. In addition, focus should be confined to the window until the user closes it with an action, such as the close button of the ‘ESC’ key. This is usually done with JavaScript to manage, confine, and direct the focus. Specific instructions can be referenced in ‘The current state of modal dialog accessibility.’ 

 **User Impact:**
When focus is not maintained during interaction with a dialog/pop-up, assistive technology users must reorient themselves and navigate back to where they were before they triggered the control to open/close the item. 

**Note:** This issue is repro through out the TSDoc URL.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/focus-order

https://user-images.githubusercontent.com/85154417/121008795-027fc980-c7b1-11eb-8a9c-8c00c97e0643.mp4
